src/DevTools/__tests__/createMockNetworkLayer.test.ts

```diff
@@ -1,20 +1,24 @@
-import { IResolvers } from "graphql-tools/dist/Interfaces"
-import { graphql } from "react-relay"
+import { createMockNetworkLayerTestMutationResultsMutation } from "__generated__/createMockNetworkLayerTestMutationResultsMutation.graphql"
+import { createMockFetchQuery } from "DevTools/createMockNetworkLayer"
+import { commitMutation, graphql } from "react-relay"
 import {
   Environment,
   fetchQuery,
   GraphQLTaggedNode,
+  Network,
+  OperationBase,
   RecordSource,
   Store,
 } from "relay-runtime"
 import { createMockNetworkLayer2 } from "../index"
+jest.unmock("react-relay")
```

Comment:
That would mean removing the global mock and going though all test files that need to have it mocked and adding `jest.mock('react-relay')` in them. I don't mind the current state of things since you get a helpful error message that tells you to unmock `react-relay` if you haven't yet.